Resources/Textures/_Moffstation/PersonalItems/Wearables/freianofvoid/Remia_Silkyflower/remiaPinInversed.rsi/meta.json

```
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
```

Consider licensing under the [latest version](https://creativecommons.org/version4/):

```suggestion
  "license": "CC-BY-SA-4.0",
```